bigdl/bigdl.sh

```diff
@@ -66,11 +66,31 @@ if [[ "${ROLE}" == "Master" ]]; then
   # will never come up.
   SPARK_NUM_EXECUTORS=$((SPARK_NUM_EXECUTORS_PER_NODE_MANAGER * CURRENTLY_RUNNING_NODEMANAGERS - 1))
 
-  cat conf/spark-bigdl.conf >> /etc/spark/conf/spark-defaults.conf
+# Check if it BigDL conf or Zoo  
+
+SPARK_BIG_DL_CONF=
+
+if [ -f conf/spark-bigdl.conf ]; then
+  SPARK_BIG_DL_CONF=conf/spark-bigdl.conf
+elif [ -f conf/spark-analytics-zoo.conf ]; then
+  SPARK_BIG_DL_CONF=conf/spark-analytics-zoo.conf
+else	
+  echo "Can't find any suitable spark config for Intel BigDL/Zoo"
```

Comment:
Nit: if you exit right here, you won't need the extra if statements below.

This is the error function we use throughout the init actions -- feel free to copy paste it at the top:

```bash
function err() {
  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $@" >&2
  return 1
}
```

And then use it like this:

```bash
err "Can't find any suitable spark config for Intel BigDL/Zoo"
```